Firebase/Core/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [changed] Removed dependency on UIKit.
```

Comment:
Might be better to say "Removed UIKit import from FIRApp.h." since the likely breakage is code with missing import's on UIKit that just happened to get it from importing FIRApp.h.

Reply:
Good idea, thanks. Done!